webview/src/setup/components/studio/Settings.tsx

```diff
@@ -30,9 +31,16 @@ export const Settings: React.FC<{
             onClick={() => setShareLiveToStudio(!shareLiveToStudio)}
             checked={shareLiveToStudio}
           >
-            Share New Experiments Live
+            Share New Experiments Live*
           </VSCodeCheckbox>
         </p>
+        <p className={styles.smallFont}>
+          *The checkbox reflects the{' '}
+          <a href="https://dvc.org/doc/user-guide/project-structure/configuration#studio">
+            studio.offline
+          </a>{' '}
+          config option.
```

Comment:
[F] I have added this small bit of text instead of trying to workout which config file(s) `studio.offline` is set in and then trying to update the appropriate level(s) of config. Much like updating the `studio.token` this button will only call `config` with the `--global` flag. If users want to save `studio.token`/`studio.offline` in project/local configs then they should be capable of managing this themselves. If this does however become an issue I will fix it retroactively.

Reply:
I've started the process of discussing with the DVC team how we can make this easier to deal with.

Reply:
Should we hint that were setting things globally in the note? For example, `The checkbox reflects the global [studio.offline](https://dvc.org/doc/user-guide/project-structure/configuration#studio) config option.`

Reply:
The text in the example isn't necessarily true. If the user has set the option at the `project` or `local` level then that option will override what we set at the global level. This is from the docs and hopefully helps to explain a bit further:

<img width="701" alt="image" src="https://github.com/iterative/vscode-dvc/assets/37993418/6157efbe-22a2-4f99-8b39-bd2fc0e9bea5">

I have only added the text so that in the (rare) case that the user has set `studio.offline=true` at the project/local level that the user will have some chance of debugging what is going on.

In order to fully explain this I think I would need to add 2 paragraphs which IMO is not necessary at this stage for onboarding. 

This issue does seem to be coming up a lot though so I think I may have to build some representation of `dvc config -l --show-origin` to add to the page.